### DISCUSSION TITLE
「开源之史」系列之一/五/四：独一无二的标识开源 —— 商标和品牌的保护极简史 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#75] [Announcements]
# 「开源之史」系列之一/五/四：独一无二的标识开源 —— 商标和品牌的保护极简史 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-03-002-open-source-trademark/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
下面呢?

作者已经在 blog 中形成了专有风格: 下面呢? 没有了-style

每次都是开头一个惊奇案件,
中间一坨艰难历程,
然后, 就没有了...

至少值得应该有:

- 类似中国案例
- 当前 FLOSS 主流应用
- 未来最好如何...

也就是 DUKU 选题时一直关注的, 任何一件具体的事儿:

- 原本如何
- 应该如何
- 最好如何

以商标法而言, 和专利类似, 只是更加一个无奈的最不坏的选择...
Adidas/Adiaos/...
李逵 撞 李鬼 的事儿, 从来没有被法律清除过,
利用视觉/文学相近, 聪明人一直在蹭流量,
其中最 恶心的就是 李鬼 注册了 李逵 商标, 自己变成正经 李逵 ,
而 李逵 本人根本无从证明自己是本人...

那么, 问题来了, 什么都有可信 区块链 来证明的 web3 世界中, 
这种情况就可以杜决嘛?

如果不行, 普通程序猿, 应该具有什么习惯来保证自己基本的形象不受侵害?
